democracy_club/apps/hermes/views.py

```diff
@@ -29,7 +29,7 @@ class PostListView(ListView):
     template_name = "hermes/post_list.html"
 
     def get_queryset(self):
-        qs = self.model.objects.published().select_related("author")
+        qs = self.model.objects.published().prefetch_related("author")
```

Comment:
`select_related` no longer works with `author` (an error suggests I can only use `category` here), but I wonder if `list_select_related` would be better?

Reply:
Disregard, I've misunderstood the use of `list_select_related`.